docs/src/prototypes/chatMessages/ChatMessageWithPopover/ReactionPopup.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
+import keyboardKey from 'keyboard-key'
+import { Popup, Menu, Reaction, ReactionProps } from '@stardust-ui/react'
+
+const getAriaLabel = ({ content: numberOfPersons, icon: emojiType }: ReactionProps) => {
+  if (numberOfPersons === 1) {
+    return `One person reacted to this message with a ${emojiType} emoji. Open menu to see person who reacted.`
+  }
+  return `${numberOfPersons} people reacted this message with a ${emojiType} emoji. Open menu to see people who reacted.`
+}
+
+class ReactionPopup extends React.Component<
+  ReactionProps & { setOpen?: React.Dispatch<React.SetStateAction<boolean>> },
+  any
+> {
+  state = {
+    open: false,
+  }
+
+  handleKeyDownOnMenu = e => {
+    if ((e.shiftKey && e.keyCode === keyboardKey.Tab) || e.keyCode === keyboardKey.Tab) {
+      this.setState({ open: false })
+    }
+  }
+
+  handleOpenChange = (e, { open }) => {
+    this.setState({ open })
+  }
+
+  render() {
+    return (
+      <Popup
```

Comment:
This is the same ReactionPopup that we use in the other Prototype, with the only difference that it does not uses the autoFocusPopupBehavior. The reason for this is, if the submenu from the action menu is opened and we hover over some reaction, the focus is moved, thus the submenu is closed. Not sure if this is the behavior we want here...